Quest System/Unity Demo/Assets/QSystem/Assets/Scripts/GUI/QGUIInventory.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class QGUIInventory : Manager<QGUIInventory>
+{
+}
```

Comment:
Komt hier nog iets of is dit dode code? Want dan kan het weg. Plaats er anders even een comment bij dat het voor in de toekomst is.